### DISCUSSION TITLE
[Authorization]:Do we have a feature where we can authorise users as per there role to see relevant areas in skywalking dashboard

### DISCUSSION THREAD

**Discussion by ccfahe** [apache/skywalking#9278] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

If a developer login he cpuld able to see the service related to him not all  infrastructure details of other servers.Admin can see all services.

Any login mechanism that can be connected to LDAP and role mechanism to group the users

### Use case

Any login mechanism that can be connected to LDAP and role mechanism to group the users

### Related issues

na

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
No, there isn't. Tenancy mechanism is not in the SkyWalking.

**Reply by ccfahe** [reply]
By any chance we are going to implement this in future releases

**Reply by Superskyyy** [reply]
> By any chance we are going to implement this in future releases

Hello there, SkyWalking is driven by the community and no one has yet to come forth and provide a proposal on implementing such features. As this is a very major one, it requires more discussions before someone can just dive into it and implement it. People are welcome to share their initial ideas and form a proposal along the way.

**Reply by wu-sheng** [reply]
@ccfahe People only ask about this, never did. So I don't know. @Superskyyy gave a clear response from a community perspective.
